modal/modal.css

```diff
@@ -0,0 +1,42 @@
+@import url(/common.css);
+@import url(/button/button.css);
+@import url(/card/card.css);
+
+.modal-container{
+    display: none; 
+    position: fixed; 
+    z-index: 1; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0,0.4);
+  }
+
+  .modal-content {
+   
+    margin: auto;
+    padding: 20px;
+    width: 30%;
+  }
+  
+  
+/* The Close Button */
+.close {
+    color: #aaaaaa;
+    float: right;
```

Comment:
Suggestion: Try not using float if you can with flex.

Reply:
yea i have removed it.